tests/src/test_projects/signed_i128/src/main.sw

```diff
@@ -42,7 +42,7 @@ fn main() -> bool {
         lower: u64::max(),
     };
 
-    res = I128::from(u128_10) / I128::from_uint(u128_lower_max_u64);
+    res = I128::from(u128_10) / I128 { underlying: u128_lower_max_u64 };
```

Comment:
Yeah I should make another PR for this. The problem with that was that it wasnt compiling because the from_uint function is private and cannot be accessed in that file

Reply:
https://github.com/FuelLabs/sway-libs/pull/60